From 2f547e33c17a9d96c098c4bbd949c19cc4d65b97 Mon Sep 17 00:00:00 2001
From: Jan Petrous <jan.petrous@nxp.com>
Date: Mon, 16 May 2022 17:27:28 +0200
Subject: [PATCH 019/102] configs: s32g: pfe: sync default port configs with
 linux

The default ethernet port map was changed to support
all ports. The EVB/EVB3 requires base board (S32GRV-PLAT)
to route there PFE_EMAC_1 and free RGMII PHY KSZ9031
on PROCEVB for GMAC (eqos).

Issue: ALB-5044
Upstream-Status: Pending 

Signed-off-by: Jan Petrous <jan.petrous@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/configs/s32g399ardb3.h | 2 +-
 include/configs/s32g3xxaevb.h  | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/configs/s32g399ardb3.h b/include/configs/s32g399ardb3.h
index d8515e8ad2..fc26b73f1b 100644
--- a/include/configs/s32g399ardb3.h
+++ b/include/configs/s32g399ardb3.h
@@ -12,7 +12,7 @@
 #define FDT_FILE			"s32g399a-rdb3.dtb"
 
 #ifdef CONFIG_FSL_PFENG
-#  define PFENG_MODE			"enable,sgmii,none,rgmii"
+#  define PFENG_MODE			"enable,sgmii,sgmii,rgmii"
 #  define PFENG_EMAC			"0"
 #endif
 
diff --git a/include/configs/s32g3xxaevb.h b/include/configs/s32g3xxaevb.h
index a84c67e530..649fe58a54 100644
--- a/include/configs/s32g3xxaevb.h
+++ b/include/configs/s32g3xxaevb.h
@@ -12,8 +12,8 @@
 #define FDT_FILE			"s32g3xxa-evb.dtb"
 
 #ifdef CONFIG_FSL_PFENG
-#  define PFENG_MODE			"enable,none,rgmii,rgmii"
-#  define PFENG_EMAC			"1"
+#  define PFENG_MODE			"enable,sgmii,sgmii,rgmii"
+#  define PFENG_EMAC			"0"
 #endif
 
 #if defined(CONFIG_USB)
-- 
2.17.1

